tests/tool/Integration.Tests/IntegrationScenarios/WebLibrary/csharp/Original/packages.config

```diff
@@ -4,5 +4,5 @@
   <package id="Microsoft.AspNet.Razor" version="3.2.7" targetFramework="net472" />
   <package id="Microsoft.AspNet.WebPages" version="3.2.7" targetFramework="net472" />
   <package id="Microsoft.Web.Infrastructure" version="1.0.0.0" targetFramework="net472" />
-  <package id="Newtonsoft.Json" version="6.0.5" targetFramework="net472" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net472" />
```

Comment:
This is the "before" project for an end-to-end test. It's meant to have out-of-date packages that UA should update. What's the reason we need to change them?

Reply:
CG alerting on this, and we need to upgrade the package versions to address the alert.

Reply:
Can we upgrade to a lower version? Seeing that UA successfully replaces old versions of packages that don't work with .NET 6 with newer versions that do work is a nice feature of this test.

Reply:
Done, picked an older version that is not listed in CG